pkgs/development/tools/misc/c2ffi/default.nix

```diff
@@ -0,0 +1,64 @@
+{ lib
+  ,stdenv
+  ,fetchFromGitHub
+  ,cmake
+  ,llvmPackages_10
+  ,llvmPackages_11
+  ,c2ffiBranch ? "11.0.0"
+}:
+
+let
+  versionSpec = {
+    "11.0.0" = {
+      version = "2021-04-15";
+      rev = "bfde21a005b7301019ece26eced1e7c12e37027e";
+      sha256 = "03q7q3l8fdjs9zs7d9l9i2wmhcixrc0lmlnxiis0apwdrgqqy028";
+      llvmPackages = llvmPackages_11;
+    };
+    "10.0.0" = {
+      version = "2021-04-12";
+      rev = "7371b79cdd2c0b9f091b345f362bc77f5e188f63";
+      sha256 = "0ldpdcdfvp27f9lbnx9xb663az2i6h0jwiq996mbpxpf2mylgy59";
+      llvmPackages = llvmPackages_10;
+    };
+  }.${c2ffiBranch};
```

Comment:
I think this should use an update script.

Reply:
@SuperSandro2000 i don't know what is that. i found this article: https://garbas.si/2016/updating-your-nix-sources.html

but there's no match for `updateFromGitHub` in the nixpkgs repo. can you please give me a keyword hint, or a URL to read?